linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml

```diff
@@ -58,6 +68,8 @@ ocil: |-
     <pre>$ grep -i upgradeRemoveDroppedPackages {{{ pkg_manager_config_file }}}</pre>
     The output should return something similar to:
     <pre>solver.upgradeRemoveDroppedPackages=true</pre>
+    {{% elif 'ubuntu' in product %}}
+    <pre>$ grep -i remove-unused {{{ pkg_manager_config_file }}}</pre>
```

Comment:
This is will render the `ocil` for Ubuntu as just the grep command. Is that intentional?

Reply:
Nice catch! 
I just pushed a fix for that.

Reply:
The first sentence will still be the `grep` command, :grin: 

Just wanted to point out that the other `ocil` start with:
`To verify that <tt>...</tt> is configured properly, run the
    following command:`